📖  add title icons list to PR template (#941)
kubernetes-sigs/cluster-api-provider-azure: .github/pull_request_template.md

 <!-- If this is your first PR, welcome! Please make sure you read the [contributing guidelines](../CONTRIBUTING.md). -->

>  <!-- please add a icon to the title of this PR (see ../CONTRIBUTING.md), and delete this line and similar ones -->

Reviewer: ```suggestion
 <!-- please add an icon to the title of this PR (see ../CONTRIBUTING.md), and delete this line and similar ones -->
```

Reviewer: ```suggestion
 <!-- please add an icon to the title of this PR (see ../CONTRIBUTING.md) -->
```
(?)
why delete the comment lines?

Author: it says that in CAPI :/
